backend/config/config.go

```diff
@@ -10,9 +10,10 @@ import (
 
 // A Config holds all configurable settings from a yml config file
 type Config struct {
-	Debug             bool   `default:"false"`
-	ListenAddr        string `default:"0.0.0.0:8080"`
-	MetricsListenAddr string `default:"127.0.0.1:9991"`
+	Debug              bool   `default:"false"`
+	ListenAddr         string `default:"0.0.0.0:8080"`
+	MetricsListenAddr  string `default:"127.0.0.1:9991"`
```

Comment:
This feels a bit weird to have a hardcoded fallback here, could we move this to the `sample.env`? Maybe throw an error is there aren't defined?

Reply:
I think it makes sense to have sane fallbacks to variables where possible, some users may be running this through docker and simply omit fields that aren't necessary to override.